src/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My To-Do list</title>
+  </head>
+  <body>
+    <header><h1>My To Do list:</h1></header>
+    <main class="container">
+      <div class="list-container">
+        <ul id="todo-list">
+          <li>
+            <p>Today's To DO</p>
+            <svg

```

Comment:
- [ ] you can add a javaScript function for the refresh button so when the user clicks here the page loads.